bears/jinja2/Jinja2Bear.py

```diff
@@ -321,6 +365,11 @@ def run(self,
             this: ``{{ var_name }}``. This can be set to any number of spaces
             via the setting ``variable_spacing``.
             Malformatted variable tags are detected and fixes suggested.
+        * Statement spacing:
+            Statement tags should be padded with one space on each side, like
+            this: ``{% statement %}``. This can be set to any number of spaces
+            via the setting ``statement_spacing``.
+            Malformatted statement tags are detected and fixes suggested.
```

Comment:
I think you can leave this sentence, it's repeated from before. I think users understand that coala could propose changes :)